doc/build/orm/loading_columns.rst

```diff
@@ -65,16 +65,18 @@ Deferred Column Loader Query Options
 ------------------------------------
 
 Columns can be marked as "deferred" or reset to "undeferred" at query time
-using options which are passed to the :meth:`_query.Query.options` method; the most
+using options which are passed to the :meth:`_sql.Select.options` method; the most
 basic query options are :func:`_orm.defer` and
 :func:`_orm.undefer`::
 
     from sqlalchemy.orm import defer
     from sqlalchemy.orm import undefer
+    from sqlalchemy import select
+
+    stmt = select(Book)
+    stmt = stmt.options(defer('summary'), undefer('excerpt'))
+    session.execute(stmt)
```

Comment:
`session.scalars(stmt).all()`

if query has all(), that returns a list, so `session.scalars(stmt).all()` will produce the same list result.